### ISSUE TITLE
Sorry, we were unable to convert your code.

### ISSUE THREAD

**Issue by rkochar** [pulumi/kube2pulumi#105]
### What happened?

I am trying out Pulumi and wanted to convert some simple kubernetes yaml files. They fail with "Sorry, we were unable to convert your code." and further down "could not read output file: : open : no such file or directory" on the browser at https://www.pulumi.com/kube2pulumi/.

### Example

This is fairly simple code. I put this into the Custom tab and clicked "Convert".

```
apiVersion: v1
kind: Namespace
metadata:
  name: "kafka"
  labels:
    name: "kafka"
```

### Output of `pulumi about`

https://www.pulumi.com/kube2pulumi/

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by mikhailshilkov** [comment]
I can reproduce your error... Interestingly, the following code works

```
apiVersion: v1
kind: Namespace
metadata:
  name: "kafka"
  labels:
    name: kafka
```

(the only difference is quotes around the label name)

We'll need to check what is going on.

**Comment by rquitales** [comment]
This issue no longer reproduces. Likely fixed by: #110
